### PR TITLE
Add API endpoint to Web Client to fetch the exclusions

### PR DESCRIPTION
**Pull request by rita-sonarsource**
Fixes https://github.com/SonarSource/sonarlint-visualstudio/issues/3042

### CODE REVIEW

File: SonarQube.Client/Api/V7_20/GetExclusionsRequest.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        internal const string GlobalExclusionsKey = "sonar.global.exclusions";
+        internal const string InclusionsKey = "sonar.inclusions";
+        internal const string GlobalInclusionsKey = "sonar.global.inclusions";
+
+        [JsonProperty("component")]
+        public virtual string ProjectKey { get; set; }
+
+        [JsonProperty("keys")]
+        public virtual string Keys { get; } =
+            string.Join(",", ExclusionsKey, GlobalExclusionsKey, InclusionsKey, GlobalInclusionsKey);
```
Comment (duncanp-sonar) [inline SonarQube.Client/Api/V7_20/GetExclusionsRequest.cs:42]: Minor: wouldn't just having a single hard-coded string be easier to read? I'm not sure the individual constants add anything except clutter.

**Review by ugras-ergun-sonarsource — APPROVED** [review]

File: SonarQube.Client.Tests/Requests/Api/V7_20/GetExclusionsRequestTests.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+}";
+
+            SetupHttpRequest(handlerMock, request, response);
+
+            var result = await testSubject.InvokeAsync(httpClient, CancellationToken.None);
+            result.Should().NotBeNull();
+
+            result.Exclusions.Should().BeEquivalentTo("**/value1", "value2", "some/value/3");
+            result.GlobalExclusions.Should().BeEquivalentTo("some/value/4");
+            result.Inclusions.Should().BeEmpty();
```
Comment (duncanp-sonar) [inline SonarQube.Client.Tests/Requests/Api/V7_20/GetExclusionsRequestTests.cs:155]: There isn't a test that checks `inclusions` correctly returns a value, is there? i.e. the `InclusionsKey` constant could be changed and all of the tests would still pass.

**Comment by duncanp-sonar** [comment]
I didn't see a separate ticket for adding a new method to `ISonarQubeService` etc. Is that part of the current ticket?

**Review by duncanp-sonar — COMMENTED** [review]
Looks good. One comment about a test, and a question about updating `ISonarQubeService`.

**Comment by rita-sonarsource** [comment]
> I didn't see a separate ticket for adding a new method to `ISonarQubeService` etc. Is that part of the current ticket?

You're right, I missed that. I'll add it in a new PR.

**Pull request merged**